lib/rouge/lexers/csvs.rb

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class CSVS < RegexLexer
+      tag 'csvs'
+      title "csvs"
+      desc 'The CSV Schema Language (http://digital-preservation.github.io/csv-schema/)'
+      mimetypes 'text/x-csvssrc'
+      filenames '*.csvs'
+
+      def self.detect?(text)
+        return true if text.shebang? 'csvs'
+      end
+
+      def self.constants
+        @constants ||= Set.new %w(nil false true)
+      end
+
+      def self.builtins
+        @builtins ||= Set.new %w(
+          args call clone do doFile doString else elseif for if list
+          method return super then
+        )
+      end
+
+      state :root do
+        rule /\s+/m, Text
+        rule %r(//.*?\n), Comment::Single
```

Comment:
Including the newlines could lead to failures to match if a comment ends a file. You don't need to include a newline, Ruby will match to the end of the line without it.